### DISCUSSION TITLE
Poke Api Error "Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'front_default')"

### DISCUSSION THREAD

**Discussion by IxshelC** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#357] [Q&A] [answered]
Me sale un error en la consola y no sé de qué es D:  Si busca los pokemones y las imágenes, pero sale ese error.
![3](https://user-images.githubusercontent.com/87400845/158003649-61cc0b6d-3895-40fb-8e9b-501f20afb653.png)
![2](https://user-images.githubusercontent.com/87400845/158003652-7cba7ae6-e4a5-475c-945d-e062d5777e06.png)
![1](https://user-images.githubusercontent.com/87400845/158003688-69194671-11ed-4efe-8f71-8d264e78b568.png)

**Comment by doguedogue** [comment, marked as answer]
comenta la linea 20 fetchPokemon()

**Reply by IxshelC** [reply]
Muchísimas gracias :')
